src/helpers/helpers.test.js

```diff
@@ -0,0 +1,15 @@
+import { getVideoId, sightGalleryDataAdapter } from "./index";
+
+describe("Check helpers func", () => {
+  test('check get video func works well', () => {
+        const url = 'https://www.youtube.com/watch?v=56bWMAEvSdA';
+        const result = getVideoId(url);
+        expect(result).toEqual('56bWMAEvSdA');
```

Comment:
Use .toEqual to compare recursively all properties of object instances

../// toMatch